packages/web-components/fast-foundation/src/slider/slider.template.ts

```diff
@@ -37,12 +37,14 @@ export function sliderTemplate(
                 </div>
                 <slot></slot>
                 <div
-                    ${ref("thumb")}
-                    part="thumb-container"
-                    class="thumb-container"
+                    ${ref("thumbPosition")}
+                    part="thumb-position"
+                    class="thumb-position"
```

Comment:
is this just the position or is it the thumb as well? Or does it contain the thumb? The position is relevant to how this is built but this element isn't necessarily *the* position, is it? I'd almost prefer to call this the `thumb` considering that follows the pattern of track above.

Reply:
I think it could be `thumb-container` but the intent is this is just the wrapper for the `thumb`. I can't think of a reason it should have any visual styling, but it should be completely separate from something like setting the `thumb` size or outline, etc. The current sample styles don't do this, but I should revisit that.

I agree with keeping the container and slots paired. In this case I think the most consistent approach there would actually be to _add_ a `thumb` wrapper element inside `thumb-position` and around the slot.

Reply:
Went back to `thumb-container` following previous logic and distinction between the actual `thumb`.